pkg/cmd/admin/router/router.go

```diff
@@ -135,7 +219,8 @@ func loadDefaultCert(file string) (string, error) {
 	return string(bytes), err
 }
```

Comment:
Any reason to duplicate code within the 2 methods - they both look like the same code (its late so I might have missed something here). Would be better to just have one method called ```loadData``` or something to that effect which has the read data + error handling code and have the other 2 methods just be ```aliases``` effectively.